jung-algorithms/src/test/java/edu/uci/ics/jung/layout/spatial/FRLayoutsTest.java

```diff
@@ -109,9 +111,11 @@ public static void check() {
     log.debug("mapTwo:{}", mapTwo);
     log.debug("mapThree:{}", mapThree);
 
-    Assert.assertTrue(
-        "the compared maps are not close enough: mapTwo:" + mapTwo + ", mapThree:" + mapThree,
-        closeEnough(mapTwo, mapThree));
+    assertWithMessage(
+            "The compared maps are not close enough: mapTwo: %s, mapThree: %s", mapTwo, mapThree)
+        .that(mapTwo)
+        .comparingValuesUsing(Correspondence.tolerance(1.0E-4))
```

Comment:
Have you had a read through [this page](https://google.github.io/truth/floating_point#for-approximate-equality-what-tolerance-should-i-use) on the Truth website about the sort of general-rule-of-thumb tolerance you should be aiming to test against? I ask because although I think that a tolerance of `1.0E-4` looks pretty reasonable, I also think you might be able to reduce it to a smaller number if the values in `mapTwo` tend to be small.

Reply:
On the viz side, they all become pixel integers, so almost any tolerance will do. I'm really just looking for cases where they are way off.

Reply:
Okay, cool, no problems for me here then. A comment would be good, I think, for future maintainers, but I won't insist. :)